src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
+        <b-container class="index-container">
+        <b-row>
+          <b-col>
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <a href="https://forms.gle/482EoBYtJozWpYCf9" class="btn" role="button" target="_blank">Apply Now</a>
+            </b-col>
+            <b-col>
+              <img src="/images/illustration.png" alt="two people sitting in front of a computer">
+          </b-col>
+          </b-row>
+        </b-container>
```

Comment:
I think you do not need to close the container here. I would just start a new row with the cards in them

Reply:
Okay, I'll try that too